### PR TITLE
Extend ruby 2.2.2

### PR DESCRIPTION
**Pull request by shivanshgaur**
Extending support to ruby version 2.2.2

### CODE REVIEW

**Comment by palkan** [comment]
Thanks!

Let's add a changelog record and we're done

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 ## master
 
+## 0.5.0 (2018-08-29)
+
+- [PR [#19](https://github.com/palkan/isolator/pull/19)] Adding support for ruby version 2.2.2. ([@shivanshgaur][])
```
Comment (palkan) [inline CHANGELOG.md]: Please, add you handle link to the end of the file to make it work

Comment (shivanshgaur) [reply CHANGELOG.md]: Done

**Comment by shivanshgaur** [comment]
can we merge this?

**Comment by palkan** [comment]
Released as 0.5.0

**Comment by shivanshgaur** [comment]
thanks

**Pull request merged**